lib/stat_tracker.rb

```diff
@@ -236,6 +240,10 @@ def fewest_tackles(season_id)
     end[0]
   end
 
+  def least_accurate_team(season_id)
+    shots_to_goals_ratio_per_team(season_id).min_by { |season, avg| avg}[0].name
+  end
+
```

Comment:
@hopesgit I'm getting an error for this method here: _NoMethodError: undefined method `name' for "6":String_

Reply:
That's weird, I had that fixed before. It got un-fixed.

Reply:
Thanks, Hope!!